tests/run-make/symbol-mangling-version/Makefile

```diff
@@ -0,0 +1,47 @@
+# ignore-cross-compile
```

Comment:
Can you rename this test directory to `symbol-mangling-hashed` to make the test name more specific to what is being tested?

Reply:
Okay, rename it `symbol-mangling-hashed`. Thank you!